Jenkinsfile

```diff
@@ -24,6 +24,9 @@ elifePipeline {
                 'test': {
                     withCommitStatus({
                         sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_app_test app npm test"
+                        sh "vault.sh kv get -field token secret/containers/elife-xpub/codecov > .codecov-token"
+                        sh "ls -l .codecov-token"
+                        sh "curl -s https://codecov.collibra.com/bash | bash -s -- -t @.codecov-token"
```

Comment:
I believe this should be ran within the container so should be part of the docker-compose file. probably https://github.com/elifesciences/elife-xpub/blob/develop/docker-compose.ci.yml as it shouldn't be needed in the production build

Reply:
Yes, it should run in the containers to provide its own reproducible environment. The file can be mounted with `volumes` in the `docker-compose.ci.yml` you mentioned.